src/pheval_exomiser/post_process/post_process.py

```diff
@@ -1,17 +1,19 @@
 from pathlib import Path
 
-from pheval_exomiser.config_parser import ExomiserConfig
 from pheval_exomiser.post_process.post_process_results_format import create_standardised_results
+from pheval_exomiser.prepare.tool_specific_configuration_options import ExomiserConfigurations
 
 
-def post_process_result_format(config: ExomiserConfig, raw_results_dir: Path, output_dir: Path):
+def post_process_result_format(
+    config: ExomiserConfigurations, raw_results_dir: Path, output_dir: Path, phenotype_only: bool
```

Comment:
Why did you decide that the `phenotype_only` should be a separate argument rather than remain in the config? Is it because this argument is also an argument on the pheval run CLI?

Reply:
No, it was because it's not part of the `ExomiserConfigurations` class which defines all the variables in the `tool_specific_configurations` field of the config. The `phenotype_only` is part of the main part of the config which is consistent in all the tool configs, so I couldn't access it from the `config: ExomiserConfigurations` parameter so I made it a separate argument

Reply:
I don't remember exactly what our discussions were wrt to that parameter and where it should live; I am sure its all correct :)